docs/blueprints/EP028.rst

```diff
@@ -0,0 +1,153 @@
+    |                     |
+    | kytos.log (file)    |
+    |                     |
+    |                     |
+    +----------^----------+
+               |
+               | write
+               |
+    +----------+----------+
+    |                     |
+    |                     |
+    |                     |
+    | RotatingFileHandler |
+    |                     |
+    | logging.ini         |
+    |                     |
+    |                     |
+    |                     |
+    |                     |
+    +---------------------+
+
+
+As represented in this diagram, as long as ``logging.ini`` has a handler that's writing to a file, Filebeat can read from it as an input, so it's up to network operators to configure it accordingly and also make sure to use the same formatter and dissect processors to parse and structure the fields of the log messages. For instance, this ``logging.ini`` below is based on `amlight/kytos-docker logging.ini <https://github.com/amlight/kytos-docker/blob/master/logging.ini>`_:
+
+.. code-block:: none
+
+    [formatters]
+    keys: file
+
+    [loggers]
+    keys: root
+
+    [formatter_file]
+    format: %(asctime)s - %(levelname)s [%(name)s] [%(filename)s:%(lineno)d:%(funcName)s] (%(threadName)s) %(message)s
+
+    [handler_file]
+    class: handlers.RotatingFileHandler
+    args:["/var/log/kytos/kytos.log", "a", 10*1024*1024, 5]
+    formatter: file
+    level: INFO
+
+    [logger_root]
+    level: INFO
+    handlers: file
+    propagate: True
+
+The following ``filebeat.yml`` configuration would correctly process, ingest and process the structured logs based on the expected placeholders of the file formatter, notice that the tokenizer with this configuration ``'%{kytos.timestamp} - %{kytos.level} [%{kytos.logger}] [%{kytos.file}:%{kytos.line}:%{kytos.func}] (%{kytos.thread}) %{kytos.message}'`` matches accordingly the expected fields in the formatter:
+
+.. code-block:: none
+
+    ---
+    filebeat.modules:
+    - module: system
+      syslog:
+        enabled: false
+      auth:
+        enabled: false
+
+    processors:
+      - dissect:
+          # kytos log formatter tokenizer to be able to search for fields easily
+          tokenizer: '%{kytos.timestamp} - %{kytos.level} [%{kytos.logger}] [%{kytos.file}:%{kytos.line}:%{kytos.func}] (%{kytos.thread}) %{kytos.message}'
+          field: "message"
+          target_prefix: ""
+
+    filebeat.inputs:
+    - type: log
+      enabled: true
+      # Paths that should be crawled and fetched. Glob based paths.
+      # Make sure not file is defined twice as this can lead to unexpected behaviour.
+      paths:
+        - /var/log/kytos/kytos.log
+
+    output.elasticsearch:
+      hosts: ${ELASTICSEARCH_HOSTS}
+      username: ${ELASTICSEARCH_USERNAME}
+      password: ${ELASTICSEARCH_PASSWORD}
+
+    setup.kibana:
+      host: ${KIBANA_HOST}
+      username: ${ELASTICSEARCH_USERNAME}
+      password: ${ELASTICSEARCH_PASSWORD}
```

Comment:
```suggestion
      password: ${ELASTICSEARCH_PASSWORD}


Points of attention
===================

After deploying such a feature, the administrator/network-operator must be aware of the following aspects:

- The Elasticsearch/Kibana web interface must be restricted to allow access only from trusted networks
- Elasticsearch must define rotation policies to avoid problems with the storage of the logs
- Whenever possible, the filebeat service architecture should provide means of running it in a different host (avoiding I/O concurrency with normal Kytos server)
```

Reply:
Great points, Italo. Let me commit this.